src/OpenTelemetry/BaseExportProcessor.cs

```diff
@@ -46,6 +46,13 @@ public sealed override void OnStart(T data)
         /// <inheritdoc />
         public abstract override void OnEnd(T data);
 
+        internal override void SetParentProvider(BaseProvider parentProvider)
```

Comment:
@cijothomas I needed a way to set the Exporter.ParentProvider when the Processor's gets set. Internal to SDK though.